git_repo/services/service.py

```diff
@@ -244,7 +244,10 @@ def format_path(self, repository, namespace=None, rw=False):
         if not rw and '/' in repo:
             return '{}/{}'.format(self.url_ro, repo)
         elif rw and '/' in repo:
-            return '{}:{}'.format(self.url_rw, repo)
+            if self.url_rw.startswith('ssh://'):
```

Comment:
in what context do you need `ssh://` URIs? As far as I can tell, it's not supported by neither Git or SSH directly (only by `xdg-` tools or browsers.